deployment/v3/utils/prop_migrator/file_comparator.py

```diff
@@ -0,0 +1,135 @@
+            for line in new_lines:
+                line = line.strip()
+                if "=" not in line:
+                    continue  # Skip lines that don't contain '='
+                key, value = line.split("=", 1)
+                new_properties[key] = value
+
+            for key, value in new_properties.items():
+                new_only_properties.append((new_filename, key, value))
+        else:
+            print(f"Old property file not found and New property file not found for {old_filename}. Skipping...")
+        continue
+
+    with open(old_file, 'r') as old:
+        old_lines = old.readlines()
+
+    with open(new_file, 'r') as new:
+        new_lines = new.readlines()
+
+    old_properties = {}
+    new_properties = {}
+
+    for line in old_lines:
+        line = line.strip()
+        if "=" not in line:
+            continue  # Skip lines that don't contain '='
+        key, value = line.split("=", 1)
+        old_properties[key] = value
+
+    for line in new_lines:
+        line = line.strip()
+        if "=" not in line:
+            continue  # Skip lines that don't contain '='
+        key, value = line.split("=", 1)
+        new_properties[key] = value
+
+    different_keys = set(old_properties.keys()) & set(new_properties.keys())
+
+    for key in different_keys:
+        old_value = old_properties[key]
+        new_value = new_properties[key]
+        if old_value != new_value:
+            different_properties.append((old_filename, new_filename, key, old_value, new_value))
+
+    old_only_keys = set(old_properties.keys()) - set(new_properties.keys())
+    for key in old_only_keys:
+        old_value = old_properties[key]
+        old_only_properties.append((old_filename, key, old_value))
+
+    new_only_keys = set(new_properties.keys()) - set(old_properties.keys())
+    for key in new_only_keys:
+        new_value = new_properties[key]
+        new_only_properties.append((new_filename, key, new_value))
+
+# Create different_properties.csv
+different_properties_csv = "different_properties.csv"
+with open(different_properties_csv, 'w', newline='') as file:
+    writer = csv.writer(file)
+    writer.writerow(["Old Property File", "New Property File", "Key", "Old Value", "New Value"])
+    for item in different_properties:
+        writer.writerow(item)
+
+print("different_properties.csv created successfully.")
+
+# Create file1_only.csv
+file1_only_csv = "file1_only.csv"
+with open(file1_only_csv, 'w', newline='') as file:
+    writer = csv.writer(file)
+    writer.writerow(["Old Property File", "Key", "Old Value"])
+    for item in old_only_properties:
+        writer.writerow(item)
+
+print("file1_only.csv created successfully.")
+
+# Create file2_only.csv
+file2_only_csv = "file2_only.csv"
+with open(file2_only_csv, 'w', newline='') as file:
+    writer = csv.writer(file)
+    writer.writerow(["New Property File", "Key", "New Value"])
+    for item in new_only_properties:
+        writer.writerow(item)
+
```

Comment:
add three CSV's in new output directory which will be created as part of this script.